templates/kernel/lifecycle.html

```diff
@@ -0,0 +1,784 @@
+              <td>Apr 2018</td>
+              <td>Apr 2020</td>
+              <td>Mar 2025</td>
+            </tr>
+            <tr>
+              <td><strong>Ubuntu 18.04.1 LTS (v4.15)</strong></td>
+              <td>Jul 2018</td>
+              <td>Apr 2020</td>
+              <td>Apr 2023</td>
+            </tr>
+            <tr>
+              <td><strong>Ubuntu 16.04.5 LTS (v4.15)</strong></td>
+              <td>Aug 2018</td>
+              <td>Apr 2020</td>
+              <td>Apr 2021</td>
+            </tr>
+            <tr>
+              <td><strong>Ubuntu 18.04.2 LTS (v4.18)</strong></td>
+              <td>Feb 2019</td>
+              <td>&nbsp;</td>
+              <td>Jul 2019</td>
+            </tr>
+            <tr>
+              <td><strong>Ubuntu 18.04.3 LTS (v5.0)</strong></td>
+              <td>Aug 2019</td>
+              <td>&nbsp;</td>
+              <td>Jan 2020</td>
+            </tr>
+            <tr>
+              <td><strong>Ubuntu 18.04.4 LTS (v5.3)</strong></td>
+              <td>Feb 2020</td>
+              <td>&nbsp;</td>
+              <td>Jul 2020</td>
+            </tr>
+            <tr>
+              <td><strong>Ubuntu 20.04.0 LTS</strong></td>
+              <td>Apr 2020</td>
+              <td>Apr 2022</td>
+              <td>Apr 2025</td>
+            </tr>
+            <tr>
+              <td><strong>Ubuntu 20.04.1 LTS</strong></td>
+              <td>Jul 2020</td>
+              <td>Apr 2022</td>
+              <td>Apr 2025</td>
+            </tr>
+            <tr>
+              <td><strong>Ubuntu 18.04.5 LTS</strong></td>
+              <td>Aug 2020</td>
+              <td>Apr 2022</td>
+              <td>Apr 2023</td>
+            </tr>
+            <tr>
+              <td><strong>Ubuntu 20.04.2 LTS</strong></td>
+              <td>Feb 2021</td>
+              <td>&nbsp;</td>
+              <td>Aug 2021</td>
+            </tr>
+            <tr>
+              <td><strong>Ubuntu 20.04.3 LTS</strong></td>
+              <td>Aug 2021</td>
+              <td>&nbsp;</td>
+              <td>Feb 2022</td>
+            </tr>
+            <tr>
+              <td><strong>Ubuntu 20.04.4 LTS</strong></td>
+              <td>Feb 2022</td>
+              <td>&nbsp;</td>
+              <td>Jul 2022</td>
+            </tr>
+            <tr>
+              <td><strong>Ubuntu 20.04.5 LTS</strong></td>
+              <td>Aug 2022</td>
+              <td>Aug 2024</td>
+              <td>Apr 2025</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+
+      <div class="p-tabs__content" id="support-all" role="tabpanel" aria-labelledby="support-all-tab">
+        <h3>Ubuntu Kernel Support</h3>
+        <div class="u-hide--small" id="kernelall"></div>
+        <p>This is a combined view of the diagrams from above (ie the Ubuntu Kernel Release Schedule combined with the Distilled 14.04.x, 16.04.x, and 18.04.x Ubuntu Kernel Support diagrams)</p>
```

Comment:
I’m sorry I didn’t pick this up when editing the copydoc earlier: Whenever you can see this text, the diagram is a combined view of some diagrams, sure — but none of them are present on the page, so they’re not “above”.

I think the message this paragraph is trying to convey is already conveyed, more concisely, by the tab selector. So the paragraph can be dropped altogether.
```suggestion
```